torch/csrc/jit/backends/backend.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
```

Comment:
minor: I think this file should appear somewhere less deeply nested, since external users will depend on it. For example, it could be something like `torch/backend.h`, analogous to `torch/custom_class.h`